Documentation/Contributing/development-flow.md

```diff
@@ -52,25 +52,26 @@ the following settings in your IDE:
 * Trim trailing whitespace
 * Markdown Table of Contents is correctly updated automatically
 
-For example, in VS Code this translates to the following settings:
+#### VS Code
+
+!!! tip
+    VS Code should prompt you automatically with some recommended extensions to install.
+    E.g., Markdown All in One, Go and YAML validator.
+
+The following settings are recommended when working on Rook:
 
 ```json
 {
-    "editor.formatOnSave": true,
-    "go.buildOnSave": "package",
-    "go.formatTool": "goreturns",
-    "files.trimTrailingWhitespace": true,
-    "files.insertFinalNewline": true,
-    "files.trimFinalNewlines": true,
-    "markdown.extension.toc.unorderedList.marker": "*",
-    "markdown.extension.toc.githubCompatibility": true,
-    "markdown.extension.toc.levels": "2..2"
+  "editor.formatOnSave": true,
```

Comment:
Instead of repeating the settings here, how about we just link to the settings.json? Actually, the tip below already has that info, so maybe we just remove this json blob here.

Reply:
Sounds good, we can even try the "insert file" functionality from the new mkdocs theme.